src/RadworksGovernor.sol

```diff
@@ -0,0 +1,232 @@
+import {ICompoundTimelock} from
+  "@openzeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol";
+import {GovernorSettings} from "@openzeppelin/contracts/governance/extensions/GovernorSettings.sol";
+
+import {IRadworksTimelock} from "src/interfaces/IRadworksTimelock.sol";
+import {GovernorTimelockCompound} from "src/lib/GovernorTimelockCompound.sol";
+import {GovernorCompatibilityBravo} from
+  "@openzeppelin/contracts/governance/compatibility/GovernorCompatibilityBravo.sol";
+
+/// @notice The upgraded Radworks Governor: Bravo compatible and built with OpenZeppelin.
+contract RadworksGovernor is
+  Governor,
+  GovernorVotesComp,
+  GovernorTimelockCompound,
+  GovernorSettings
+{
+  struct ProposalVote {
+    uint256 againstVotes;
+    uint256 forVotes;
+    uint256 abstainVotes;
+  }
+
+  /// @notice The address of the RAD token on Ethereum mainnet from which this Governor derives
+  /// delegated voting weight.
+  ERC20VotesComp private constant RAD_TOKEN =
+    ERC20VotesComp(0x31c8EAcBFFdD875c74b94b077895Bd78CF1E64A3);
+
+  /// @notice The address of the existing Radworks DAO Timelock on Ethereum mainnet through
+  /// which
+  /// this Governor executes transactions.
+  IRadworksTimelock private constant TIMELOCK =
+    IRadworksTimelock(payable(0x8dA8f82d2BbDd896822de723F55D6EdF416130ba));
+
+  /// @notice Human readable name of this Governor.
+  string private constant GOVERNOR_NAME = "Radworks Governor Bravo";
+
+  /// @notice The number of RAD (in "wei") that must participate in a vote for it to meet quorum
+  /// threshold.
+  ///
+  /// TODO: placeholder
+  uint256 private constant QUORUM = 100_000e18; // 100,000 RAD
+
+  /// @param _initialVotingDelay The deployment value for the voting delay this Governor will
+  /// enforce.
+  /// @param _initialVotingPeriod The deployment value for the voting period this Governor will
+  /// enforce.
+  /// @param _initialProposalThreshold The deployment value for the number of RAD required to
+  /// submit
+  /// a proposal this Governor will enforce.
+  constructor(
+    uint256 _initialVotingDelay,
+    uint256 _initialVotingPeriod,
+    uint256 _initialProposalThreshold
+  )
+    GovernorVotesComp(RAD_TOKEN)
+    GovernorSettings(_initialVotingDelay, _initialVotingPeriod, _initialProposalThreshold)
+    GovernorTimelockCompound(TIMELOCK)
+    Governor(GOVERNOR_NAME)
+  {}
+
+  /**
+   * @dev Mapping from proposal ID to vote tallies for that proposal.
+   */
+  mapping(uint256 => ProposalVote) private _proposalVotes;
+
+  /**
+   * @dev Mapping from proposal ID and address to the weight the address
+   * has cast on that proposal, e.g. _proposalVotersWeightCast[42][0xBEEF]
+   * would tell you the number of votes that 0xBEEF has cast on proposal 42.
+   */
+  mapping(uint256 => mapping(address => uint256)) private _proposalVotersWeightCast;
+
+  /// @dev We override this function to resolve ambiguity between inherited contracts.
```

Comment:
on this overridden function (and the others further down), perhaps it'd be useful to @inheritdoc the natspec of the source

Reply:
Done 4c8b4aa5079ff2ca8a0124eeab3624cc5cfce0db